src/Symfony/Component/Ldap/Security/LdapBadge.php

```diff
@@ -75,6 +79,8 @@ public function getQueryString(): ?string
 
     public function markResolved(): void
     {
+        trigger_deprecation('symfony/ldap', '6.4', 'Calling %s is deprecated.', __METHOD__);
```

Comment:
Should we add more explanation, than just "... is deprecated." Maybe sth. like:
"... is deprecated and not used for the LdapBadge anymore because of foo?"

Reply:
Updated, thank you